urbanopt-scenario-gem.gemspec

```diff
@@ -27,11 +28,13 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '~> 13.0'
   spec.add_development_dependency 'rspec', '~> 3.7'
 
-  spec.add_dependency 'json-schema'
-  spec.add_dependency 'json_pure'
-  spec.add_dependency 'openstudio-common-measures', '~> 0.2.0'
-  spec.add_dependency 'openstudio-model-articulation', '~> 0.2.0'
-  spec.add_dependency 'sqlite3', '1.4.2'
-  spec.add_dependency 'urbanopt-core', '~> 0.3.0'
-  spec.add_dependency 'urbanopt-reporting', '~> 0.1.0'
+  spec.add_runtime_dependency 'json-schema', '~> 2.8'
+  spec.add_runtime_dependency 'json_pure', '~> 2.3'
+  spec.add_runtime_dependency 'openstudio-common-measures', '~> 0.2'
```

Comment:
we need another digit on the openstudio/urbanopt gems...i'm not comfortable allowing upgrades on such a wide range.  thanks!